### PR TITLE
Rewrite the NApp.

### PR DESCRIPTION
**Pull request by diraol**
Fix kytos/of_topology#5

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -26,104 +23,64 @@ def execute(self):
...
+        msg = 'The MAC %s is reachable from switch/port %s/%s.'
+        log.debug(msg, ethernet.source, event.source.switch.id,
+                  event.message.in_port)
 
-    @staticmethod
     @listen_to('kytos/of_core.v0x0[14].messages.in.ofpt_port_status')
-    def update_port_stats(event):
-        """Receive a Kytos event and update port.
+    def update_port_status(self, event):
+        """Dispatch 'port.[created|modified|deleted]' event.
```
Comment (cemsbr) [inline main.py]: We have to think of a way to automatically detect event names to ease documentation.

Comment (beraldoleal) [reply main.py]: @cemsbr sure, that is the idea.. but only after the next release :)

**Comment by beraldoleal** [comment]
@diraol Can you please add a very hightlight warning on each napp readme, saying that this is a experimental NApp and changes will come ?

Because, I will accept this PR but probably we will change some events soon.

**Comment by diraol** [comment]
Ok @beraldoleal Will do it right now.

**Comment by diraol** [comment]
@beraldoleal warning added at #10

**Pull request merged**